Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1014,6 +1014,8 @@ public abstract partial class BaseMeshSync : MonoBehaviour, IObservable<MeshSync
 
         MaterialHolder dst = m_materialList.Find(a => a.id == materialID);
 
+        bool isNewMaterial = dst == null || dst.material == null;
+        
         //if (invalid && creating materials is allowed)
         if ((dst == null || dst.material == null || dst.name != materialName) && importerSettings.CreateMaterials) {
             if (null == dst) {
```

Comment:
Perhaps it's better to remove line 1031
`dst.ShouldApplyMaterialData = false;`

as well ?

Reply:
Yeah, removed now